ui/lib/kv/addon/components/page/secret/metadata/details.hbs

```diff
@@ -17,7 +17,7 @@
 <h2 class="title is-5 has-bottom-padding-s has-top-margin-l">
   Custom metadata
 </h2>
-<div class="box is-fullwidth is-sideless is-paddingless is-marginless">
+<div class="box is-fullwidth is-sideless is-paddingless is-marginless" data-test-kv-metadata-section>
```

Comment:
this is technically the `custom-metadata` section, right?

Reply:
oh yeah, wrong copypaste